CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ the code was deployed.
 
 ### Fixes
 - Now always gets the most recent unresponded session (CU-tb9pvc).
+- Fixed built in documentation in add_hubs.sh script(CU-faxfbg)
```

Comment:
Can you please add a space before the opening parenthesis and a period on the end of the line?